src/pages/cli/migration/lambda-node-version-update.mdx

```diff
@@ -1,73 +0,0 @@
-export const meta = {
```

Comment:
is there a directory.js entry for this page that should also be removed?

Reply:
oh good callout! yes there was an entry, and it has since been removed :)